plugins/SitesManager/templates/sites-list/site-fields.html

```diff
@@ -31,7 +31,9 @@
             <ul>
                 <li>
                     <span class="title">{{ 'SitesManager_Urls'|translate }}</span>:
-                    {{ site.alias_urls.join(', ') }}
+                    <div = ng-repeat = "url in site.alias_urls">
```

Comment:
Some more points: 
* It does not show the comma between urls anymore.
* Can you add a target=_blank so the links open in a new window?
* Also add `rel=noreferrer` to respect privacy and not expose Piwik URL
* Instead of `href` use `ng-href` so link will be only working once actually angular is rendered